src/index.html

```diff
@@ -10,8 +10,197 @@
+        
+        <div>
+          <label for="terms">I accept the term of the agreement</label>
+          <input
+            type="checkbox" 
+            id="terms" 
+            name="terms"
+            required
+          >
+        </div>
+        
+      </fieldset>
+
+      <fieldset class="field_style">
+        <legend>Registration</legend>
+
+        <div class="input_style">
+          <label for="email">E-mail:</label>
+          <input
+            type="email" 
+            id="email" 
+            name="email" 
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+        
+        <div>
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            minlength="8"
+            maxlength="32"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="field_style">
+        <legend>An interesting fact about you!</legend>
+      
+        <div class="input_style">
+          Do you love cats?
+        
+          <input
+            type="radio"
+            name="aboutCatYes"
+            id="aboutCatYes"
+          >
+        
+          <label for="aboutCatYes">Yes</label>
+      
+          <input
+            type="radio"
+            name="aboutCatNo"
+            id="aboutCatNo"
+          >
+
+          <label for="aboutCatNo">No</label>
+        
+        </div>
+      
+        <div class="input_style">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            id="color"
+            name="color"
+            value="#000000"
+          >
+        </div>
+      
+        <div class="input_style">
+          <label for="time">What time do you go to bed?</label>
+          <input
+            type="time"
+            id="time"
+            name="time"
+          >
+        </div>
+      
+        <div class="input_style">
+          <label for="car">What are your favorite brand of cars?</label>
+          <select
+            name="chooseCar" 
+            id="car"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+      
+        <div>
+          <label for="rate">How do you rate our work?</label>
+          <input
+            type="range"
+            id="rate"
+            name="rate"
+            min="1"
+            max="10"
+            step="1"
+            value="6"
+          >
+        </div>
+      
+      </fieldset>
+    
+      <fieldset class="field_style">
+        <legend>Additional info:</legend>
+        
+        <div class="input_style">
+          <label for="comments">Comments:</label>
+          <textarea
+            name="comments"
+            id="comments">
+          </textarea>
```

Comment:
and this
<img width="280" alt="image" src="https://user-images.githubusercontent.com/102411130/218978066-16b27fcf-0b75-4af0-a8b7-c58f3c558969.png">
<img width="888" alt="image" src="https://user-images.githubusercontent.com/102411130/218978119-6a311a1f-185a-42e0-9640-eae27f949415.png">